custom/ewsghana/api.py

```diff
@@ -332,11 +335,25 @@ def _set_up_supply_point(self, location, ews_location):
             location.name = supply_point.name
             location.site_code = supply_point.code
             location.location_type = supply_point.type
-            self._create_supply_point_from_location(supply_point, location)
+            try:
+                sql_location = SQLLocation.objects.get(domain=self.domain, site_code=supply_point.code)
```

Comment:
There are some districts that have the same site code as supply points. I agreed with Mike that they are more important for supply points.